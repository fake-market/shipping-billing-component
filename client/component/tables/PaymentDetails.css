:local(.payment) {
  border: 1px solid black;
  font-size: 18px;
  font-family: helvetica, arial, sans-serif;
  width: 100%;
}

th {
  height: 40px;
}

td {
  height: 30px;
}
