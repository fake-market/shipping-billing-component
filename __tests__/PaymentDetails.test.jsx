import React from 'react';
import Enzyme, {shallow, mount, render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { configure } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';

import PaymentDetails from '../client/component/tables/PaymentDetailsTable';

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

configure({ adapter: new Adapter() });

describe('Component: PaymentDetailsTable', () => {
  const wrapper = shallow(<PaymentDetails />);

  it('PaymentDetailsTable should be defined', () => {
    expect(wrapper).toBeDefined();
  });

  it('PaymentDetailsTable should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('PaymentDetailsTable should render table', () => {
    expect(wrapper.find('table').length).toBe(1);
  });
});
