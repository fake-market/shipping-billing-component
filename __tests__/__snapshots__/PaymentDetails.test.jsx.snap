// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: PaymentDetailsTable PaymentDetailsTable should render correctly 1`] = `
<div>
  <h3>
    Payment details
  </h3>
  <table>
    <thead>
      <tr>
        <th>
          Payment Method
        </th>
        <th>
          Preferred / Accepted
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          <img
            src="test-file-stub"
          />
        </td>
        <td>
          PayPal Preferred
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
